cloud-etl/docs/index.rst

```diff
@@ -0,0 +1,254 @@
+*  Span multiple cloud providers (AWS, GCP, Azure) and on-prem datacenters
+*  Use Kafka to aggregate data in single source of truth
+*  Harness the power of `KSQL <https://www.confluent.io/product/ksql/`__
+
+
+End-to-end Streaming ETL
+========================
+
+This demo showcases an entire end-to-end cloud ETL deployment, built for
+100% cloud services:
+
+-  Kinesis source connector: reads from a Kinesis stream and writes the
+   data to a Kafka topic in |ccloud|
+-  KSQL: streaming SQL engine that enables real-time data processing
+   against Kafka
+-  GCS or S3 sink connector: pushes data from Kafka topics to cloud
+   storage
+
++-----------------------+-----------------------+---------------------+
+| Component             | Consumes From         | Produces To         |
++=======================+=======================+=====================+
+| Kinesis source        | Kinesis stream        | Kafka topic         |
+| connector             | ``demo-logs``         | ``eventLogs``       |
++-----------------------+-----------------------+---------------------+
+| KSQL                  | ``eventLogs``         | KSQL streams and    |
+|                       |                       | tables              |
+|                       |                       | (`ksql.commands     |
+|                       |                       | <ksql.commands>`__) |
++-----------------------+-----------------------+---------------------+
+| GCS/S3/Blob sink      | KSQL tables           | GCS/S3/Blob         |
+| connector             | ``COUNT_PER_SOURCE``, |                     |
+|                       | ``SUM_PER_SOURCE``    |                     |
++-----------------------+-----------------------+---------------------+
+
+Warning
+=======
+
+This demo uses real cloud resources, including that of |ccloud|,
+Amazon Kinesis, and one of the cloud storage providers. To avoid
+unexpected charges, carefully evaluate the cost of resources before
+launching the demo and ensure all resources are destroyed after you are
+done running it.
+
+Prerequisites
+=============
+
+Cloud services
+--------------
+
+-  `Confluent Cloud cluster <https://confluent.cloud>`__: for development only. Do not use a production cluster.
+-  `Confluent Cloud KSQL <https://docs.confluent.io/current/quickstart/cloud-quickstart/ksql.html>`__ provisioned in your |ccloud|
+-  Download `Confluent Platform <https://www.confluent.io/download/>`__ |release|: for more advanced Confluent CLI functionality (optional)
+-  AWS or GCP or Azure access
+
+Local install
+-------------
+
+-  `Confluent Cloud CLI <https://docs.confluent.io/current/quickstart/cloud-quickstart/index.html#step-2-install-the-ccloud-cli>`__
+   v0.234.0 or later
+-  ``aws`` CLI, properly initialized with your credentials: AWS Kinesis and (optional) if destination is AWS S3
+-  ``gsutils`` CLI, properly initialized with your credentials: (optional) if destination is GPC GCS
```

Comment:
``gsutil`` ? (no 's'). I tested this only because I have it locally but my run thorugh will go to aws

Reply:
Good catch!!